subcommand/webhook-cert-manager/command.go

```diff
@@ -59,6 +62,10 @@ func (c *Command) init() {
 	c.flagSet = flag.NewFlagSet("", flag.ContinueOnError)
 	c.flagSet.StringVar(&c.flagConfigFile, "config-file", "",
 		"Path to a config file to read webhook configs from. This file must be in JSON format.")
+	c.flagSet.StringVar(&c.flagDeploymentName, "deployment-name", "",
+		"Name of deployment that is the owner of the secret")
```

Comment:
Not clear what "the secret" is. I think we can just say name of the deployment this pod is running in. What we do with that is an impl detail

Reply:
done